README.md

```diff
@@ -29,3 +29,30 @@ seat.to # parent node. nil for the final node
 seat.position # node position id
 seat.payload # custom payload that can be also seeded via BracketGraph::Graph#seed
 ```
+
+## Double Elimination Winner Bracket
```

Comment:
This should be "Double Elimination Bracket" (it contains both winner and loser groups)